src/swarm/node/model/NeoNode.d

```diff
@@ -805,10 +802,12 @@ public class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
             credentials = &s.cred;
         }
 
+        auto no_delay = true;
+
         // Instantiate params object shared by all neo connection handlers.
         auto neo_conn_setup_params = new Neo.ConnectionHandler.SharedParams(
             options.epoll, options.shared_resources, options.requests,
-            options.no_delay, *credentials, this, &this.getResourceAcquirer);
+            no_delay, *credentials, this, &this.getResourceAcquirer);
```

Comment:
Should the `SharedParams` field also be flagged in some way? e.g. a comment saying to remove it in the next major.

Reply:
Good point :+1:

Reply:
Or maybe a deprecated ctor.

Reply:
Actually, I can rip out the fair chunk of this flag, just keeping the classes exposed to the *protos intact.

Reply:
I've pushed another commit ripping it all out.

Reply:
Looks like the interface is intact, it still compiles with the protos.